dashboard2/publishwebsite/javascript-publishing.md

```diff
@@ -0,0 +1,61 @@
+# JavaScript publishing
+
+This section contains key settings for [Basic publishing](clientsidetranslation.html) as well as the one line of JavaScript to embed the translations to the original website. There are four sub-sections.
+
+## Embed code
+
+This part is the most important for the publishing process. You can embed the client-side translator engine via one line of JavaScript code that you can find here. Use the *Copy to clipboard* button, then add it to the website.
+
+![Embed code screen](/img/dashboard2/crest-settings/embed_code.png)
+
+## Tweaks
+
+These checkboxes are fairly similar to those that you can find on the *Advanced settings* section. They influence the behaviour of the translator script.
+
+Note that the majority of these tweaks change the embed code, so after changing them, please make sure to update the code on the website.
+
+![Tweaks screen](/img/dashboard2/crest-settings/tweaks.png)
+
+There are seven options:
+
+- *Include all the default parameters*: By default, the embed code is kept as short as possible. To achieve this, tweaks that aren't enabled aren't included in the URL. This tweak allows you to include them all.
+- *Language parameter*: By default, you can change the target language by appending `?__ptLanuguage=${target_language}` to the URL of the site. With this option, you can change it from `__ptLanguage`.
+- *Storage parameter*: By default, the user's language choice is stored in LocalStorage under `ptStorage`. Change it with this option.

```

Comment:
is it `__` or no `__`? The placeholder in the input field disagrees with this now...

Reply:
That's very much a mistake on my end. Docs and your above comment says no `__`. It also says that it should be `ptLanguage`, not `ptStorage` which is the part I didn't notice.

I realise it's different on the Dashboard, I changed it there too, but I'm yet to update the screenshot.